### PR TITLE
Remove FirstTest-Typespec

### PR DESCRIPTION
**Pull request by mcgallan**
# Description
<i>Add your description here!</i>

# Checklist

To ensure a quick review and merge, please ensure:
- [ ] The PR has a understandable title and description explaining the _why_ and _what_.
- [ ] The PR is opened in draft if not ready for review yet.
   - If opened in draft, please allocate sufficient time (24 hours) after moving out of draft for review
- [ ] The branch is recent enough to not have merge conflicts upon creation.

# Ready to Land?
- [ ] Build is completely green
   - Submissions with test failures require tracking issue and approval of a CODEOWNER
- [ ] At least one +1 review by a CODEOWNER
- [ ] All -1 reviews are confirmed resolved by the reviewer 
   - Override/Marking reviews stale must be discussed with CODEOWNERS first

### CODE REVIEW

File: test/CadlRanchMockApis/src/FirstTest-TypeSpec.ts [hunk 1]
```diff
@@ -42,59 +42,3 @@ Scenarios.FirstTest_CreateLiteral = passOnSuccess([
     };
   }),
 ]);
```
Comment (ArcturusZhang) [inline test/CadlRanchMockApis/src/FirstTest-TypeSpec.ts:44]: any reason why we have to keep this scenario here?

Comment (mcgallan) [reply test/CadlRanchMockApis/src/FirstTest-TypeSpec.ts]: fixed

File: test/CadlRanchProjects.Tests/FirstTest-TypeSpec.cs [hunk 2]
```diff
@@ -1,132 +0,0 @@
...
-        /*
-        [Test]
-        public async Task FirstTest_ObjectAsHeader() => await Test(async (host) =>
-        {
-            Response response = await new FirstTestTypeSpecClient(host).ObjectAsHeaderAsync(new HeadModel("key1", "key2"));
-            Assert.AreEqual(204, response.Status);
-        });
-        */
-        [Test]
-        public void TestToObjectFromJson()
```
Comment (pshao25) [inline test/CadlRanchProjects.Tests/FirstTest-TypeSpec.cs:125]: This is no E2E test. Find who added this from history and confirm if it is still needed.

File: test/CadlRanchMockApis/src/FirstTest-TypeSpec.ts [hunk 3]
```diff
@@ -1,100 +0,0 @@
-import { passOnSuccess, ScenarioMockApi, mockapi, json } from "@azure-tools/cadl-ranch-api";
-import { ValidationError } from "@azure-tools/cadl-ranch-api";
-
-/**
- * Test mock server for `FirstTest-TypeSpec` test project.
- */
-export const Scenarios: Record<string, ScenarioMockApi> = {};
-
-Scenarios.FirstTest_CreateLiteral = passOnSuccess([
```
Comment (pshao25) [inline test/CadlRanchMockApis/src/FirstTest-TypeSpec.ts:9]: So, 6 operations are deleted. You should delete the corresponding 6 operations in test\TestProjects\FirstTest-TypeSpec\FirstTest-TypeSpec.tsp

**Review by pshao25 — CHANGES_REQUESTED** [review]
Please do as commented.

File: test/CadlRanchProjects.Tests/FirstTest-TypeSpec.cs (hunk 2 above)
Comment (mcgallan) [reply test/CadlRanchProjects.Tests/FirstTest-TypeSpec.cs:125]: fixed

**Review by pshao25 — APPROVED** [review]

**Pull request merged**